src/js/components/List/List.js

```diff
@@ -52,6 +52,10 @@ const StyledList = styled.ul`
     props.itemFocus &&
```

Comment:
This is looking great, although it looks like this block from props.itemFocus is causing the focus indicator to still appear when interacting with mouse. I don't believe this block is necessary anymore given the latest focus/unfocus styles. Can you remove this block entirely? I checked on Chrome, Safari, and Firefox and it behaved as expected for mouse + keyboard navigation:

comment + code to be removed:
```
 // during the interim state when a user is holding down a click,
  // the individual list item has focus in the DOM until the click
  // completes and focus is placed back on the list container.
  // for visual consistency, we want to keep the focus indicator on the
  // list container the whole time.
  ${(props) =>
    props.itemFocus &&
    focusStyle({ forceOutline: true, skipSvgChildren: true })}}
```